# Dumb plan
# Every three numbers sets a motion for a period in time and is executed sequentially:
# [linear velocity, angular velocity, duration (seconds)]

dumb_plan: [0, 0, 10,
            5, 0, 3,
            5, 0, 3,
            -3, 0, 3              
            0, 0, 1]

# .5 0
# 1 .021
# 1.5 .189
# 2 .402
# 2.5 .630
# 3 .970
# 3.5 1.389
# 4 2.052
# 4.2 2.554
# 4.4 2.996
# 4.6 3.622
# 4.8 4.074
# 5  4.571

# Hardware PID
kp_vel: 1
ki_vel: 0
kd_vel: 0

kp_turn: 1000
ki_turn: 0
kd_turn: 0

# roll, pitch, yaw
# float[9]
imu_orientation_covariance: [0, 0, 0,
                             0, 0, 0,
                             0, 0, 0]

# roll, pitch, yaw
# float[9]
imu_angular_velocity_covariance: [0, 0, 0,
                                  0, 0, 0,
                                  0, 0, 0]

# x, y, z
# float[9]
imu_linear_acceleration_covariance: [0, 0, 0,
                                     0, 0, 0,
                                     0, 0, 0]

# x, y, z, roll, pitch, yaw
# float[36]
twist0_covariance: [2, 0, 0, 0, 0, 0,
                    0, .5, 0, 0, 0, 0,
                    0, 0, .5, 0, 0, 0,
                    0, 0, 0, .5, 0, 0,
                    0, 0, 0, 0, .5, 0,
                    0, 0, 0, 0, 0, .5]